### ISSUE TITLE
Unit testing

### ISSUE THREAD

**Issue by anweiss** [opencontrol/oscalkit#10]
Tracking issue for missing unit tests.

**Comment by asadullah-yousuf-10p** [comment]
We do have some unit tests now in `generator/generator_test`.

**Comment by asadullah-yousuf-10p** [comment]
@anweiss some unit tests are now failing (probably after #40) due to the usage of JSON profiles which are seemingly outdated as compared to XML ones.

**Comment by anweiss** [comment]
@asadullah-yousuf-10p we can comment out the failing JSON unit tests for now. Will create a separate issue for that.
